Request PID 1209:2609 for 0x33.board MIDI keyboard

s-ol opened the pull request:
The documentation page `https://s-ol.nu/0x33.board` links out to the open-source hardware design, firmware repository, and documentation sources. The project has been submitted to OSHWA for certification.

s-ol commented:
The documentation page isn't very visual atm, so here's some proof of life:

https://twitter.com/S0lll0s/status/1543664256494850050
https://twitter.com/S0lll0s/status/1511653215984570371
https://twitter.com/S0lll0s/status/1526603234059685888

s-ol commented:
The test failure is due to #754 and doesn't seem to be related to my additions.

jepler commented:
Your keyboard looks so cool!

I checked the firmware URL at https://git.s-ol.nu/hw/0x33.board/firmware.git with a browser and it gvies an internal server error.  If I instead attempt to use it as a clone URL, I get an error from git:
```
jepler@rat:~/src$ git clone https://git.s-ol.nu/hw/0x33.board/firmware.git
Cloning into 'firmware'...
remote: counting objects: 146, done.
Receiving objects: 100% (146/146), 235.18 KiB | 650.00 KiB/s, done.
cd firmware.git
warning: remote HEAD refers to nonexistent ref, unable to checkout.
```

Please feel free to ping me once you've got the source code hosting details worked out!

s-ol commented:
Hey @jepler, thanks for taking a look. I fixed the remote HEAD and the repo should now check out correctly. Before it would've been necessary to do a `git checkout main` after cloning, but that wasn't intended.

To see the code online, there is a separate link listed on the documentation page that leads to a web-based git browser: https://git.s-ol.nu/hw/0x33.board/firmware/-/ (note the `/-/` suffix, not `.git` as for cloning).

I also rebased this branch onto master so you have a fast-forward.

jepler reviewed (approved):
Thank you! I can access the firmware & design files now.
